apps/monitoring/aat/base/kustomization.yaml

```diff
@@ -8,8 +8,6 @@ resources:
   - prometheus-values.yaml
   - monitoring-values.yaml
   - botkube-values.yaml
-  - oneagent-values.yaml
   - list-all-pods-clusterrole.yaml
 
-patchesStrategicMerge:
-  - ../../dynatrace/aat/oneagent.yaml
\ No newline at end of file
+patchesStrategicMerge:
```

Comment:
Can you fix this, the floating empty patches section (can you check the perftest one as well please)

Reply:
Removed.